packages/core/data-transfer/types/remote/protocol/client/transfer/pull.d.ts

```diff
@@ -3,14 +3,23 @@ import { CreateTransferMessage, TransferAssetFlow } from './utils';
 
 export type TransferPullMessage = CreateTransferMessage<
   'step',
-  | TransferStepCommands<'entities', IEntity>
-  | TransferStepCommands<'links', ILink>
-  | TransferStepCommands<'configuration', IConfiguration>
-  | TransferStepCommands<'assets', TransferAssetFlow | null>
+  | TransferStepCommands<'entities', IEntity[]>
```

Comment:
Do we want to enforce batching or should we accept either an array or a single value?

Either way, could you update [this line in the docs](https://github.com/strapi/strapi/blob/75275de7f5e35431a002d6228f57544fb4e20b4f/docs/docs/docs/01-core/data-transfer/02-providers/05-remote-strapi/01-websocket.md#L41) to mention that it expects an array / batch of data please? I just merged it this afternoon and now it's not entirely correct 😆